Frends.FixedWidthFlatFile.Parse/Frends.FixedWidthFlatFile.Parse/Definitions/ColumnSpecification.cs

```diff
@@ -3,22 +3,36 @@
 
 namespace Frends.FixedWidthFlatFile.Parse.Definitions;
 
+/// <summary>
+/// Column specification parameters.
+/// </summary>
 public class ColumnSpecification
 {
     /// <summary>
     /// Column name. If input data contains Header row and value is left empty, header value is used as name.
     /// </summary>
+    /// <example>Example</example>
     [DisplayFormat(DataFormatString = "Text")]
-    public string Name { get; set; }
+    public string Name { get; set; } = "";
+
+    /// <summary>
+    /// Column type.
+    /// </summary>
+    /// <example>String</example>
```

Comment:
This example should be ColumnType.String so it's specified that it's internal enum.